src/modules/main/store/mainSlice.ts

```diff
@@ -77,7 +79,7 @@ export const fetchExperts = (): AppThunkType => async (dispatch) => {
 export const fetchNewestPosts = (): AppThunkType => (dispatch, getState) => {
   const { meta } = getState().main.newest;
 
-  const totalNewestPosts =  meta.totalNewestPosts || MOCK_NEWEST.length;
+  const totalNewestPosts = meta.totalNewestPosts || MOCK_NEWEST.length;
```

Comment:
no mock needed

Reply:
My section is important. Do I have to delete it for newest?